### PR TITLE
optimize: add table meta checker switch

### PR DESCRIPTION
**Pull request by l81893521**
### Ⅰ. Describe what this PR did
Configuring table meta checker is done. But I m wondering is it necessary configuring the TABLE_META_CHECKER_INTERVAL as well?

### Ⅱ. Does this pull request fix one issue?

#1483

### CODE REVIEW

File: server/src/main/resources/file.conf [hunk 1]
```diff
@@ -118,6 +118,8 @@ transaction {
   #schedule delete expired undo_log in milliseconds
   undo.log.delete.period = 86400000
   undo.log.table = "undo_log"
+  #schedule check table meta
```
Comment (slievrly) [inline server/src/main/resources/file.conf]: This is a client configuration item.

Comment (l81893521) [reply server/src/main/resources/file.conf]: Thanks, I m changed

Comment (slievrly) [reply server/src/main/resources/file.conf]: > Thanks, I m changed

this file.conf located in server module, but server not use this configuration item.

Comment (l81893521) [reply server/src/main/resources/file.conf]: I move it to seata-config-core\src\main\resources\file.conf

**Comment by l81893521** [comment]
> add this configuration item in nacos-config.txt

ok

**Pull request merged**